### ISSUE TITLE
Cannot edit a picture (alt text etc.) if the name has umlauts in its name

### ISSUE THREAD

**Issue by AndreasSchantl** [statamic/cms#2009]
## Bug Description
A picture cannot be opened if the name contains umlauts/non latin letters.

## How to Reproduce
1) Upload an image with a name containing non latin letters.
Here to copy and paste for you to reproduce: hässlicher_bug

2) Use the image somewhere (in my case a bard set)

3) Save the entry and reload.

4) Click the edit icon and see the issue happening

## Extra Detail

![image](https://user-images.githubusercontent.com/9936988/86169526-0f645980-bb1a-11ea-9502-149c58ff2459.png)
![image](https://user-images.githubusercontent.com/9936988/86169543-17bc9480-bb1a-11ea-887b-b455289fc7d1.png)

![image](https://user-images.githubusercontent.com/9936988/86169496-03789780-bb1a-11ea-9a81-bf0fc15ec1d8.png)

## Environment

**Statamic version:** 3.0.0.-beta-31

**PHP version:** 7.4, Laravel Valet

**Install method (choose one):**
- Used V3 install

**Comment by AndreasSchantl** [comment]
I thought this has been fixed by @damcclean's #2892 but it somehow isn't. I'll probably check on this later next week if I get the time to.
